xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -542,75 +526,42 @@ bool CEdl::ReadBeyondTV(const std::string& strMovie)
 
 bool CEdl::ReadPvr(const CFileItem &fileItem)
 {
-  const std::string strMovie = fileItem.GetDynPath();
-  if (!CServiceBroker::GetPVRManager().IsStarted())
-  {
-    CLog::Log(LOGERROR, "%s - PVR Manager not started, cannot read Edl for %s", __FUNCTION__, CURL::GetRedacted(strMovie).c_str());
-    return false;
-  }
-
-  std::vector<PVR_EDL_ENTRY> edl;
-
-  if (fileItem.HasPVRRecordingInfoTag())
+  std::vector<Cut> cutlist = PVR::CPVREdl::GetCuts(fileItem);
+  for (auto& cut : cutlist)
```

Comment:
const?

Reply:
For both vector and loop?

Reply:
Does not compile without modifying method "AddCut" called further down, but might be worthwhile.

Reply:
`AddCut` does modify the cut handed over to it, so `const` does actually not work here.